upstream/CVE-2021-25741.json

```diff
@@ -0,0 +1,66 @@
+{
+  "id": "CVE-2021-25741",
+  "modified": "2021-09-13T20:58:56Z",
+  "published": "2021-09-13T20:58:56Z",
+  "summary": "Symlink Exchange Can Allow Host Filesystem Access",
+  "details": "A security issue was discovered in Kubernetes where a user may be able to create a container with subpath volume mounts to access files \u0026 directories outside of the volume, including on the host filesystem.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.19.6"
+            },
+            {
+              "introduced": "1.20.0"
+            },
+            {
+              "fixed": "1.20.11"
+            },
+            {
+              "introduced": "1.21.0"
+            },
+            {
+              "fixed": "1.21.5"
+            },
+            {
+              "introduced": "1.22.0"
+            },
+            {
+              "fixed": "1.22.1"
```

Comment:
```suggestion
              "fixed": "1.22.2"
```

Reply:
1.22.1 contains this fix - https://github.com/aquasecurity/vuln-list-k8s/pull/87#discussion_r1343585113